src/datagrid/DataGrid.stories.tsx

```diff
@@ -29,9 +30,39 @@ import {
     pageFilterFunction,
     hideableColumns,
     hideShowColFooter,
+    selectedRows,
+    getRowData,
 } from "./DataGridValues";
 import {CustomFilter} from "./CustomFilter";
+import {DataGridRow} from "./DataGrid";
 
+const store = new Store({
```

Comment:
In storybook, can we add selected row count with pagination ?

Reply:
@payald I have done that. If you see above screen shot. Out of 3 selected rows one is at second page and others are at first page.